### PR TITLE
Rename ansible-base to ansible-core for 2.11

### PR DESCRIPTION
**Pull request by etcet**
ansible-base has been renamed to ansible-core in version 2.11.

https://github.com/conda-forge/ansible-base-feedstock/issues/10
https://docs.ansible.com/ansible-core/2.11/porting_guides/porting_guide_core_2.11.html

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by etcet** [comment]
I'm not quite sure what to do here. The windows build is failing because:

`error: can't copy 'build\lib\ansible\module_utils\ansible_release.py': doesn't exist or not a regular file`
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=317290&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=3225

I believe this is because ansible_release.py is a symlink:
```
$ ls -al lib/ansible/module_utils/ansible_release.py 
lrwxrwxrwx 1 chris chris 13 Apr 26 13:53 lib/ansible/module_utils/ansible_release.py -> ../release.py
```

@conda-forge/staged-recipes is this a common issue? Is there a way to resolve the failed Windows build?

**Comment by mariusvniekerk** [comment]
We might be able to move ansible-core to a noarch package and move arch dependent deps into a metapackage this depends on?

**Comment by etcet** [comment]
I wanted to keep this as a noarch package because `ansible-base` is. But, it can't build on windows because of a symlink in the source. `ansible-base` did not build on Windows before it was added (https://github.com/conda-forge/staged-recipes/pull/12721). Also, even if it would build, it "cannot run on a Windows host natively" (https://docs.ansible.com/ansible/latest/user_guide/windows_faq.html#can-ansible-run-on-windows).

@conda-forge/staged-recipes ready for review

**Comment by tschoonj** [comment]
Can this PR be merged in please? I am seeing [failures](https://github.com/conda-forge/molecule-docker-feedstock/pull/2) due to ansible-core not being available.

File: recipes/ansible-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9470ceb710711eacd75f1bcae802af036ab2a6c2cd5a13bfcb50b2d89667c9c5
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
```
Comment (xhochy) [inline recipes/ansible-core/meta.yaml:16]:
```suggestion
  noarch: python
```

File: recipes/ansible-core/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  host:
+    - pip
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4
```
Comment (xhochy) [inline recipes/ansible-core/meta.yaml:21]:
```suggestion
    - python >=3.5
```

File: recipes/ansible-core/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  run:
+    - cryptography
+    - jinja2
+    - packaging
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4
```
Comment (xhochy) [inline recipes/ansible-core/meta.yaml:26]:
```suggestion
    - python >=3.5
```

**Review by xhochy — COMMENTED** [review]
It's OK to be noarch and then fail on Win here.

**Review by xhochy — APPROVED** [review]
👍

**Pull request merged**